price-feeder/oracle/provider/kraken.go

```diff
@@ -570,11 +535,6 @@ func (p *KrakenProvider) setCandlePair(candle KrakenCandle) {
 	p.candles[candle.Symbol] = candleList
 }
 
-// ping to check websocket connection.
-func (p *KrakenProvider) ping() error {
```

Comment:
It was used in the "handleWebSocketMsgs" method before to call the keepReconnecting method if a ping was unsuccessful. Because I removed that logic including it, the golangci-lint was catching that the ping method was implemented but not getting used. Should I readd it and just log an error if a ping is unsuccessful in that same place it was in before?